src/domain/app/BrowserApp.tsx

```diff
@@ -19,11 +19,14 @@ import browserRouter from './routes/browserRouter';
 import { CookieConfigProvider } from '../../common/components/cookieConfigProvider';
 import KukkuuHDSLoginProvider from '../auth/KukkuuHDSLoginProvider';
 import ProfileProvider from '../profile/ProfileProvider';
+import AppConfig from './AppConfig';
 
-// TODO maybe: Variables for these:
 const matomoInstance = createMatomoInstance({
-  urlBase: 'https://analytics.hel.ninja/',
-  siteId: 56,
+  urlBase: AppConfig.matomoBaseUrl,
+  siteId: +AppConfig.matomoSiteId,
```

Comment:
The plus forces a number interpretation? I don't know yet how I feel about it :D I would probably self use some kind of parser or casting option.

Reply:
i think its more like a myth that you cannot safely use it in js, you can, there are different parsing results though in some cases compared to parseInt. https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which
I think i had similar discussion with Kari.